test/acceptance/NavigationSugar.scala

```diff
@@ -46,6 +46,10 @@ trait NavigationSugar extends WebBrowser with Eventually with Assertions with Ma
     webDriver.findElement(locator).click()
   }
 
+  def verifySize(selectorId: String, expectedSize: Int)(implicit webDriver: WebDriver) = {
+    webDriver.findElements(By.cssSelector(s"[$selectorId] > li")).size() shouldBe expectedSize
+  }
```

Comment:
Perhaps `verifyListSize` would be more descriptive?  `verifySize` of a selector could be the characters, number of children (p, li, etc).